presto-oracle/src/main/java/com/facebook/presto/plugin/oracle/OracleClient.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.oracle;
+
+import io.airlift.log.Logger;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import oracle.jdbc.driver.OracleDriver;
+
+import com.facebook.presto.plugin.jdbc.BaseJdbcClient;
+import com.facebook.presto.plugin.jdbc.BaseJdbcConfig;
+import com.facebook.presto.plugin.jdbc.JdbcConnectorId;
+import com.facebook.presto.spi.SchemaTableName;
+import com.facebook.presto.spi.type.Type;
+import com.google.common.collect.ImmutableSet;
+
+public class OracleClient extends BaseJdbcClient
+{
+    private static final Logger LOG = Logger.get(OracleClient.class);
+
+    @Inject
+    public OracleClient(JdbcConnectorId connectorId, BaseJdbcConfig config, OracleConfig oracleConfig)
+            throws SQLException
+    {
+        super(connectorId, config, "", new OracleDriver());
+        connectionProperties.setProperty("nullCatalogMeansCurrent", "false");
```

Comment:
I think all of these properties are for MySQL and not Oracle. I checked the Oracle JDBC docs and it doesn't seem to have any properties that we want or need to set. So, all of these and the `OracleConfig` class can be removed.